cmd/boulder-janitor/main_test.go

```diff
@@ -0,0 +1 @@
+package main
```

Comment:
What is the purpose of this file?

Reply:
I was going to put tests here before I pulled out all the useful-to-test code into `janitor.go`. I'll remove it.

Reply:
Removed in 1ba1874